packages/terra-card/.npmignore

```diff
@@ -0,0 +1,6 @@
+*.log
```

Comment:
The draw back to the new pull request is losing all the current comments, questions, and discussions in a consolidated source. The old branch could be renamed but as long as people are able to flip back and fourth from the link above it shouldn't cause too much confusion.